src/argilla/client/client.py

```diff
@@ -208,6 +217,143 @@ def get_workspace(self) -> str:
         """
         return self._client.headers.get(WORKSPACE_HEADER_NAME)
 
+    def list_workspaces(self) -> List[WorkspaceModel]:
+        """Lists all the availble workspaces for the current user.
+
+        Returns:
+            A list of `WorkspaceModel` objects, containing the workspace
+            attributes: name, id, created_at, and updated_at.
+        """
+        return workspaces_api.list_workspaces(client=self._client.httpx).parsed
+
+    def list_datasets(self) -> List[FeedbackDatasetModel]:
+        """Lists all the available `FeedbackTask` datasets in Argilla.
+
+        Returns:
+            A list of `FeedbackDatasetModel` objects, containing the dataset
+            attributes: id, name, workspace_id, guidelines, status, created_at, and last_updated.
+        """
+        return datasets_api_v1.list_datasets(client=self._client.httpx).parsed
+
+    def create_dataset(self, name: str, workspace_id: str, guidelines: Optional[str] = None) -> FeedbackDatasetModel:
+        """Creates a new `FeedbackTask` dataset in Argilla.
+
+        Args:
+            name: The name of the dataset.
+            workspace_id: The id of the workspace in which the dataset will be created.
+            guidelines: The guidelines of the dataset. Defaults to `None`.
+
+        Returns:
+            A `FeedbackDatasetModel` object, containing the dataset attributes: id, name,
+            workspace_id, guidelines, status, created_at, and last_updated.
+        """
+        return datasets_api_v1.create_dataset(
+            client=self._client.httpx,
+            name=name,
+            workspace_id=workspace_id,
```

Comment:
Since all these methods are referring the new FeedbackDataset and are used in the `feedback.py`, I would move to the module as private functions, for 2 main reasons:

1. Keep current/old code as invariant as possible -> (new things in a different place will help us with the migration progress)
2. *Dataset* is a concept already defined in Argilla, so can be confusing having a `list_datasets` method returning only V1 ones.

Reply:
Yes, I remember you mentioned this some days ago, can we schedule a quick call to see the approach you're thinking of? Thanks 😄